sdks/go/sdk_test.go

```diff
@@ -39,14 +39,14 @@ func TestSDK(t *testing.T) {
 	assert.False(t, sm.shutdown)
 	assert.False(t, sm.hm.healthy)
 
-	s.Ready()
+	_ = s.Ready()
```

Comment:
I feel like these should actually be

```go
err := s.Ready() # change
assert.Nil(t, err) # add this line
assert.True(t, sm.ready)
assert.False(t, sm.shutdown)
```

For all the below. Better to fix the test, since we're here (no idea why I didn't do this in the first place)

Reply:
Sorry for this. Totally missed it too.